resources.md

```diff
@@ -0,0 +1,59 @@
+1. GITHUB (https://github.com/drupal-ukraine)
+  - Andrii Podanenko (https://www.facebook.com/podarok)
```

Comment:
We have a link with a list of admins: 
https://github.com/orgs/drupal-ukraine/people?query=role%3Aowner

The same with FB groups:
facebook Ukraine - https://www.facebook.com/groups/drupal.ua/admins/
facebook Lutsk - https://www.facebook.com/groups/DrupalCafeLutsk/admins/

I quess, it willl be btter to use them. This will be hard and useless to update this document once new admins added. And, we will always forgetting to do that (update the cocument).